ReactWindows/ReactNative/UIManager/DependencyObjectExtensions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using ReactNative.Bridge;
+using System;
+using System.Runtime.CompilerServices;
+using Windows.UI.Xaml;
+
+namespace ReactNative.UIManager
+{
+    static class DependencyObjectExtensions
+    {
+        private static readonly ConditionalWeakTable<DependencyObject, DependencyObjectData> s_properties =
+            new ConditionalWeakTable<DependencyObject, DependencyObjectData>();
+
+        public static void SetTag(this DependencyObject view, int tag)
```

Comment:
Not for now, let's minimize the diff.

---
In reply to: [68079207](https://github.com/ReactWindows/react-native-windows/pull/487#discussion_r68079207) [](ancestors = 68079207)